src/script/components/ParticipantItemContent/ParticipantItemContent.tsx

```diff
@@ -39,50 +39,48 @@ import {
 } from './ParticipantItem.styles';
 
 export interface ParticipantItemContentProps {
-  name: string;
+  renderParticipantBadges?: (user: User) => React.ReactNode;
```

Comment:
This is the main change. Since we would need to inject both the `user` and the `conversation` through props drilling, it's easier here to give a `renderProps` and the top level parent is responsible for the logic of displaying the verification badges